### DISCUSSION TITLE
Want to change the color of the text column description

### DISCUSSION THREAD

**Discussion by 0xyzone** [filamentphp/filament#11879] [Help]
### Package

Table builder

### Package Version

v3.2

### How can we help you?

I am trying to change the color of description of the text column
`TextColumn::make('orderItems.item.name')
                                ->description('Items', position: 'above')
                                ->listWithLineBreaks()
                                ->extraAttributes([
                                    'class' => 'my-4'
                                ]),`
How can i achieve that?

![image](https://github.com/filamentphp/filament/assets/58982560/4e5746cd-ffd9-4a62-aec6-f0904b65525f)
Want that in color white

**Comment by leandrocfe** [comment]
```php
->description(fn () => new HtmlString('<span class="text-white">Items</span>'), 'above')
```
?
